### DISCUSSION TITLE
How to drop images onto your gist?

### DISCUSSION THREAD

**Discussion by egberts** [community/community#23311] [New to GitHub]
<p>I am trying to add a JPEG image into my new gist (on <a href="https://gist.github.com/" rel="nofollow noopener">https://gist.github.com/</a>) so that I can in turn use these images in my README.md and GitHub repository’s Wiki page(s).</p>
<p>But it doesn’t seem to work on Linux Debian 11 Firefox 73.0.1.</p>
<p>The only thing that works is absolute URL path.</p>
<p>Is this image drop into Gist supposed to work?</p>

**Comment by francisfuzz** [comment, marked as answer]
<p>Hi there, <a class="mention" href="/egberts">@egberts</a>––sorry for the late reply here!</p>
<p>It’s possible to drag and drop textual files to your Gist, but not images at this time.</p>
<p>As a workaround, you could open an issue or pull request and drag and drop the image as a file attachment:</p>
<p><a href="https://help.github.com/en/github/managing-your-work-on-github/file-attachments-on-issues-and-pull-requests" rel="nofollow noopener">https://help.github.com/en/github/managing-your-work-on-github/file-attachments-on-issues-and-pull-requests</a></p>
<p>This produces an image URL that you can use in one of your gists (since Markdown is supported).</p>
<p>I hope this helps as a workaround!</p>
